lib/ansible/modules/system/sysvinit.py

```diff
@@ -301,7 +301,13 @@ def main():
         def runme(doit):
 
             args = module.params['arguments']
-            cmd = "%s %s %s" % (script, doit, "" if args is None else args)
+            last_cmd = '%s %s' % (doit, "" if args is None else args)
+
+            if location.get('service'):
+                # standard tool that has been 'destandarized' by reimplementation in other OS/distros
```

Comment:
this is exactly why we don't use it as the primary method of dealing with init scripts, in many cases it won't use sysvinit but other service managers